solr/test-framework/build.gradle

```diff
@@ -43,6 +43,17 @@ dependencies {
   var zkExcludes = {
     exclude group: "org.apache.yetus", module: "audience-annotations"
   }
+  api('org.apache.curator:curator-client', {
```

Comment:
Debatable if "api" is appropriate here

Reply:
I've changed all curator deps here to "implementation"